app/_how-tos/block-unauthorized-requests-with-opa.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Block unauthorized requests in {{site.base_gateway}} with the OPA plugin
+content_type: how_to
+
+products:
+    - gateway
+
+tier: enterprise
+
+works_on:
+    - on-prem
+    - konnect
+
+min_version:
+  gateway: '3.4'
+
+plugins:
+  - opa
+
+entities:
+  - service
+  - route
+  - plugin
+
+tags:
+    - security
+
+tldr:
+    q: How can I set up a policy with OPA in {{site.base_gateway}}?
+    a: Run an OPA server and create a policy, then enable the OPA plugin and specify the `config.opa_host` and `config.opa_path` parameters.
+      
+
+tools:
+    - deck
+
```

Comment:
```suggestion
related_resources:
  - text: Allow clients to choose their authentication methods and prevent unauthorized access
     url: /how-to/allow-multiple-authentication/

```
I thought users might like to see some related how tos about authorization

Reply:
I don't think this is really related, OPA doesn't deal with authentication